### ISSUE TITLE
hashtags search not working

### ISSUE THREAD

**Issue by Triden** [mgp25/Instagram-API#877]
I'm sorry for the duplicate. You just did not respond. I read a little information. There are suggestions that the application writes photos exif. Now when you create a personal image there will not be recorded exif. I really would like to resolve the problem with the search. Do you have at least some suggestions?

**Comment by Triden** [comment]
Well, where are all gone?) It's very important that the posts are on the hashtag.)

**Comment by GuilhermeFM** [comment]
I will give you a sugestion. when posting something, the API chooses one url that comes from a request made before sending the post image or video. If you stop to analyse this API you will see that it always choose the same url in the response my guess is maybe the app is internally doing something to choose the url that will not filter your post.

And you will ask me, how do i know whats de app is doing ? Best answer is: do some reversing into the app and try to find out.

**Comment by Triden** [comment]
I know that the service was able to fix it. Only he Russian. smmplanner.

**Comment by Grebennik** [comment]
Got the same problem.
If i have added hashtag, i can find it only with web version client or from account, who is following account with post with hashtag.

And from common mobile client - i can't see post, if i'm not follow account with post with hashtag.
-----
P.S. it writing to me, that there are six posts for example (with this hashtag), but showing only 4 from other users.

**Comment by Triden** [comment]
Maybe there is not enough of a parameter for the transfer? Maybe he thinks that at this moment the account is hidden privacy settings, this does not show the photos in the public domain?
https://help.instagram.com/273447449440719

**Comment by Grebennik** [comment]
It seems to be truth.

**Comment by Triden** [comment]
I find out what the video is! Perhaps the puzzle is not so complicated ??? For those who do not understand what the problem is. If you use this API to make a post, the other accounts will not be able to find your records by hashtag. What's so have a photo that is not present in the video? It is also important!

**Comment by Taytus** [comment]
You are not following the template for issues nor have provided ANY code. You are NOT entitled to a reply if you don't even care about following very basic guidelines.

**Comment by Triden** [comment]
How would you show the code? The original code of the library? You yourself try. Make publication with hashtag. On the other account, locate it on the hashtag. Accounts should not be signed at each other.

**Comment by Triden** [comment]
[https://github.com/mgp25/Instagram-API/issues/874](#874l)

**Comment by ping** [comment]
@Triden as you have linked in #874 yourself, mgp25 has already told you that this is a *known problem* without a solution so far. Your theories about comments and exif can easily be tested yourself but you don't seem to have made any effort to do so. The API already supports comment posting. And if you believe exif helps, use any exif editor to edit a photo manually and upload the photo through the API. 

Coming here to repeatedly say that it's "important" is clearly not very productive isn't it?

**Comment by Triden** [comment]
Sorry, I did not speak very well English. I wrote here, because my knowledge is not enough to solve the problem. I would try, if I could get the query text from the official application to the latest version of the placement office. If you have helped, I would have tried to find out. I have installed a sniffer, but could not decipher https. They were able to accommodate any exif or something inside. It is necessary to get a picture of the request and view other options. If possible, send me a request for a formal application.
